src/Microsoft.AspNetCore.BuildTools.ApiCheck/Loader/AssemblyLoader.cs

```diff
@@ -1,33 +1,30 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Linq;
+#if NET452
+using System.IO;
+#endif
 using System.Reflection;
 #if NETCOREAPP1_1
-using NuGet.Frameworks;
 using NuGet.ProjectModel;
-#endif
 using NugetReferenceResolver;
-using System.IO;
+#endif
 
 namespace ApiCheck
 {
     public abstract class AssemblyLoader
     {
-        public static Assembly LoadAssembly(
-                string assemblyPath,
-                string assetsJson,
-                string framework)
+        public static Assembly LoadAssembly(string assemblyPath, string assetsJson)
         {
-            
 #if NETCOREAPP1_1
             var lockFile = new LockFileFormat().Read(assetsJson);
-            var graph = PackageGraph.Create(lockFile, framework);
+            var graph = PackageGraph.Create(lockFile, "netcoreapp1.1");
```

Comment:
Let me clarify: I didn't mean we upgrade the tool to _run_ on netcoreapp2.0. Rather I was asking if we can allows this tool to parse the graph for multiple versions of netcoreapp1.0. 

More context: some other tools hardcode netcoreapp1.0 which led to #121. Hardcoding netcoreapp1.1 is repeating the same mistake. A fix for #121 will likely revert the removal of framework if you commit this as is.